extensions/ql-vscode/src/packages/commands/CommandManager.ts

```diff
@@ -10,6 +10,7 @@ import type { Disposable } from "./Disposable";
 /**
  * A command function is a completely untyped command.
  */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I couldn't find a good way around this so ended up just ignoring the eslint error. Admittedly I didn't try super hard, but I didn't want to end up with a solution that was unnecessarily complex either.